src/lib/Scenes/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/AuctionArtworkGrid.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box } from "@artsy/palette"
+import { AuctionArtworkGrid_artwork } from "__generated__/AuctionArtworkGrid_artwork.graphql"
+import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import _ from "lodash"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ContextGridCTA } from "../../ContextGridCTA"
+import { Header } from "../../Header"
+
+interface AuctionArtworkGridProps {
+  artwork: AuctionArtworkGrid_artwork
+}
+
+export class AuctionArtworkGrid extends React.Component<AuctionArtworkGridProps> {
+  render() {
+    const {
+      artwork: { sale },
+    } = this.props
+
+    if (!sale) {
```

Comment:
technically we should never get into this `if` right?

Reply:
It would be weird but we could, especially depending on where this fragment container is consumed.